Sources/Property.swift

```diff
@@ -640,4 +635,42 @@ public final class MutableProperty<Value>: MutablePropertyProtocol {
 	deinit {
 		observer.sendCompleted()
 	}
+
+	/// Binds a signal to a target, updating the target's value to the latest
+	/// value sent by the signal.
+	///
+	/// - note: The binding will automatically terminate when the target is
+	///         deinitialized, or when the signal sends a `completed` event.
+	///
+	/// ````
+	/// let property = MutableProperty(0)
+	/// let signal = Signal({ /* do some work after some time */ })
+	/// property <~ signal
+	/// ````
+	///
+	/// ````
+	/// let property = MutableProperty(0)
+	/// let signal = Signal({ /* do some work after some time */ })
+	/// let disposable = property <~ signal
+	/// ...
+	/// // Terminates binding before property dealloc or signal's
+	/// // `completed` event.
+	/// disposable.dispose()
+	/// ````
+	///
+	/// - parameters:
+	///   - target: A target to be bond to.
+	///   - signal: A signal to bind.
+	///
+	/// - returns: A disposable that can be used to terminate binding before the
+	///            deinitialization of the target or the signal's `completed`
+	///            event.
+	@discardableResult
+	public static func <~ (target: MutableProperty, signal: Signal<Value, NoError>) -> Disposable? {
```

Comment:
Assuming https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3162 get a pass, `DynamicProperty` would have its own `<~` implementation. This as the default implementation is only useful if we keep the current easy-to-leak-because-of-the-stealth-retain `DynamicProperty`.